content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md

```diff
@@ -0,0 +1,175 @@
+
+**Junya:** Why did you decide to participate in the Kubernetes Upstream Training?
+
+**Yoshiki:** At the time I participated, I was still a student.
+I had only briefly worked with EKS, but I thought Kubernetes seemed complex yet cool, and I was casually interested in it.
+Back then, OSS felt like something out of reach, and upstream development for Kubernetes seemed incredibly daunting.
+While I had always been interested in OSS, I didn't know where to start.
+It was during this time that I learned about the Kubernetes Upstream Training and decided to take the challenge of contributing to Kubernetes.
+
+**Junya:** What were your impressions after participating?
+
+**Yoshiki:** I found it extremely valuable as a way to understand what it's like to be part of an OSS community.
+At the time, my English skills weren't very strong, so accessing primary sources of information felt like a big hurdle for me.
+Kubernetes is a very large project, and I didn't have a clear understanding of the overall structure, let alone what was necessary for contributing.
+The upstream training provided a Japanese explanation of the community structure and allowed me to gain hands-on experience with actual contributions.
+Thanks to the guidance I received, I was able to learn how to approach primary sources and use them as entry points for further investigation, which was incredibly helpful.
+This experience made me realize the importance of organizing and reviewing primary sources, and now I often dive into GitHub issues and documentation when something piques my interest.
+As a result, while I am no longer contributing to Kubernetes itself, the experience has been a great foundation for contributing to other projects.
+
+**Junya:** What areas are you currently contributing to, and what are the other projects you're involved in?
+
+**Yoshiki:** Right now, I'm no longer working with Kubernetes, but instead, I'm a maintainer of PipeCD, a CNCF Sandbox project.
+PipeCD is a CD tool that supports GitOps-style deployments for various application platforms.
+The tool originally started as an internal project at CyberAgent.
+With different teams adopting different platforms, PipeCD was developed to provide a unified CD platform with a consistent user experience.
+Currently, it supports Kubernetes, AWS ECS, Lambda, Cloud Run, and Terraform.
+
+**Junya:** What role do you play within the PipeCD team?
+
+**Yoshiki:** I work full-time on improving and developing Kubernetes-related features within the team.
+Since we provide PipeCD as a SaaS internally, my main focus is on adding new features and improving existing ones as part of that support.
+In addition to code contributions, I also contribute by giving talks at various events and managing community meetings to help grow the PipeCD community.
+
+**Junya:** Could you explain what kind of improvements or developments you are working on with regards to Kubernetes?
```

Comment:
I guess this Question and its Answer can be better combined with previous question and its answer.

Reply:
That’s a good point. 

However, I’m not a big fan of paragraphs becoming too long — especially in interviews. 
Since this isn’t a major change, I’d like to hear another opinions from other reviewers as well. 

Thank you for your comment!